lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -14,6 +24,7 @@ import SubmitGradeForm from './SubmitGradeForm';
  * @prop {ClientRPC} clientRPC - Service for communicating with Hypothesis client
  * @prop {string} courseName
  * @prop {string} assignmentName
+ * @prop {Function} onError TODO TYPING
```

Comment:
I need some advice on how to type this function property. It's literally `handleError` from `LTIBasicLaunchApp` and I am loathe to fully reproduce the typing/params here.